src/Turtle/Prelude.hs

```diff
@@ -575,6 +588,44 @@ touch file = do
 #endif
         else output file empty )
 
+type Permissions = Filesystem.Permissions -> Filesystem.Permissions
```

Comment:
The fully expanded type is just too long for my taste (especially with qualified names) and it caused me to split type signatures into multiple lines to fit 80 characters, so I introduced a synonym. I can remove it, though, if you think it's confusing.